WorkWithFiles/work_files.rb

```diff
@@ -51,7 +41,7 @@ def delete(file_name)
         rescue ArgumentError => e
```

Comment:
ArgumentError is inherited from StandardError, so you do not need to indicate both, its enough just to put   
   rescue StandardError => e
Here is link with some useful info and the The Exception Tree